src/api/comments.js

```diff
@@ -0,0 +1,29 @@
+const BASE_URL = 'https://mate-api.herokuapp.com';
+
+export function getPostComments(postId) {
+  return fetch(`${BASE_URL}/comments/`)
+    .then(response => response.json())
+    .then(result => result.data)
+    .then(comments => (
+      comments.filter(comment => comment.postId && comment.postId === postId)
+    ));
+}
+
+export function addComment(postId, name, email, body) {
```

Comment:
```suggestion
export function addComment(post) {
```
pass single object instead of separate values

Reply:
Чого так?